loop/e2e-test/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "loop-e2e-test",
+  "version": "1.0.0",
+  "description": "End-2-end test for Hello Firefox (Loop)",
+  "scripts": {
+    "pocket": "marionette-mocha --host marionette-firefox-host --timeout 60s --runtime /Volumes/FirefoxNightly.app/Contents/MacOS/firefox-bin tests/loop_signup.js",
```

Comment:
change `"pocket"` to `"loop"` at the start of L6?
Also, for `"test"` below, should that say `"./run.sh"` instead of `run-tests.sh`?